mage/query-modules/python/kmeans-clustering.md

```diff
@@ -0,0 +1,171 @@
+---
+id: kmeans-clustering
+title: kmeans_clustering
+sidebar_label: kmeans_clustering
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const Highlight = ({children, color}) => (
+<span
+style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+{children}
+</span>
+);
+
+[![docs-source](https://img.shields.io/badge/source-node2vec-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/blob/main/python/kmeans.py)
+
+## Abstract
+
+The KMeans algorithm clusters given data by trying to separate samples in `n` groups of equal variance, by minimizing a criterion known as 
```

Comment:
Please choose how we are going to call this algorithm as I can see there are a lot of variants (kMeans, k-means, K-means, K-Means) and stick to it :)

Reply:
I will stick to` k-means`